src/lib/FlightTasks/tasks/ManualPositionSmoothVel/FlightTaskManualPositionSmoothVel.hpp

```diff
@@ -65,7 +65,8 @@ class FlightTaskManualPositionSmoothVel : public FlightTaskManualPosition
 private:
 
 	enum class Axes {XY, XYZ};
-	void reset(Axes axes);
+	void reset(Axes axes, bool force_z_zero =
```

Comment:
Can you reformat this to the usual method documentation to avoid the weird line-break?
```suggestion
	 /** 
	  * Reset the required axes. when force_z_zero is set to true, the z derivatives are set to sero and not to the estimated states
	  */
	void reset(Axes axes, bool force_z_zero = false);
```